karaf/jetty-config/src/main/resources/jetty.xml

```diff
@@ -20,47 +20,4 @@
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_0.dtd">
 
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
```

Comment:
I'm not sure; I think it might break if we remove it.  Note that you can't configure the ServletContext / WebAppContext (where SessionHandler is set) at this jetty server-wide level.